File: src/altair_cpp_tests/src/custom_msg_test.cpp
#include "altair_cpp_tests/custom_msg_test_node.hpp"



int main(int argc, char *argv[]) {
    rclcpp::init(argc, argv);
    auto node = std::make_shared<altair::CustomMsgTestNode>();
    
    rclcpp::spin(node);
    rclcpp::shutdown();

    return 0;
}

File: src/altair_py_tests/altair_py_tests/custom_msg_test.py
import rclpy
from .modules.custom_msg_test_node import CustomMsgTestNode



def main(args=None) -> None:
    rclpy.init()
    node = CustomMsgTestNode()

    rclpy.spin(node)

    node.destroy_node()
    rclpy.shutdown()



if __name__ == '__main__':
    main()